lib/MT/Component.pm

```diff
@@ -758,6 +758,29 @@ sub __deep_localize_labels {
     }
 }
 
+sub __deep_localize_templatized_values {
+    my ( $c, $hash ) = @_;
+    foreach my $k ( keys %$hash ) {
+        if ( ref( $hash->{$k} ) eq 'HASH' ) {
+            __deep_localize_templatized_values( $c, $hash->{$k} );
+        }
+        else {
+            next unless $k =~ m/\A(?:
+                  name
+                | description
+                | category_field
+                | category_set
+                | content_type
+                | date_field
```

Comment:
@charsbar 
datetime_field?

Reply:
oops, thanks.

Reply:
@miniuchi
https://github.com/movabletype/movabletype/blob/develop/lib/MT/ContentData.pm#L792-L805
 and possibly https://github.com/movabletype/movabletype/blob/develop/lib/MT/Template/Tags/ContentType.pm#L287-L305 , though the latter doesn't try to load by name.